Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/managed/CertficateBean.java

```diff
@@ -160,11 +160,13 @@ public void importFileUpload(FileUploadEvent event) {
             importCertificate = new ArrayList<>();
             Certificate cert = service.createCertificate(importCertFile.getContents());
             cert.setAlias(ALIAS_PLACEHOLDER);
-            importCertificate.add(cert);
             checkCertValidity(cert);
             if (cert.getExpiresInDays() > 30 && cert.getExpiresInDays() <= 90) {
```

Comment:
Optional: Suggest to move this "expire soon" logic into the if (!expiredCert) to reduce unnecessary check if we won't allow user to import expired cert